src/Adapter/Module/AdminModuleDataProvider.php

```diff
@@ -277,6 +285,42 @@ public function setActionUrls(ModuleCollection $modules, ?string $specific_actio
         return $modules;
     }
 
+    /**
+     * @param ModuleCollection $modules
+     *
+     * @return ModuleCollection
+     */
+    public function setAdditionalDescriptions(ModuleCollection $modules): ModuleCollection
+    {
+        $modulesAdditionalDescriptions = $this->hookManager->exec('actionGetModulesAdditionalDescription', [
```

Comment:
I'm sorry but I don't see why you're using a hook? Why not doing what @PululuK said?

Reply:
It's not the first time that my point of view has been ignored.
My opinion is irrelevant in this project I think.
I'm just a poor external maintainer.

Reply:
@PululuK there's no difference between external and internal maintainers, I already heard the same kind of comment from a PrestaShop's employee 😄

I know everyone is not comfortable with the "request changes" to block the PR in order to be sure to discuss your idea, but sometimes it's worth it 🙂.

Reply:
:D @PululuK your POV is not ignored at all. Ok, I added a extraDescription field but my need is that MBO fills the extra description, not only the module itself. So we need a hook to allow this

If you look well, with what I did, the module can add an extraDescription itself but another module can add extraDescription in order to be displayed in front of each module in the ModuleManager

Believe me, I'm always happy to do what others contributors/maintainers think if it's in the interest of the project

Reply:
Maybe you guys don't understand the need. Please read aganin the issue and ask for answers ;)

Reply:
What I don't understand is why you're not doing it like it's done for the module description (or even displayName) itself:
In the `class/module/Module.php` file you have the `description` property.
In the `src/Adapter/Module/Module.php` file you can have a `description` key in the `$attribute` property.
Either the module by itself has the property `description` filled, either you can fill it from an API [like we did for ps_distributionapiclient](https://github.com/PrestaShop/ps_distributionapiclient/blob/main/src/DistributionApi.php#L81).

Also, be careful, it seems [there's already an `$additional_description` property in the `src/classes/Module.php`](https://github.com/PrestaShop/PrestaShop/blob/develop/classes/module/Module.php#L95)

Reply:
in ps_distributionapiclient, `getModulesList` is an hook action doesn't it ?